docs/design/event_and_latency_definitions/publisher.md

```diff
@@ -77,3 +77,8 @@ See also
 - [Trace points | dds_write](../trace_points/runtime_trace_points.md#ros2_caretdds_write)
 - [Trace points | bind_addr_to_addr](../trace_points/runtime_trace_points.md#ros2_caretdds_bind_addr_to_addr)
 - [Trace points | bind_addr_to_stamp](../trace_points/runtime_trace_points.md#ros2_caretdds_bind_addr_to_stamp)
+
+<prettier-ignore-start>
+!!! Info
+        If it is not after jazzy, it will not appear in generic_publish().
```

Comment:
In humble, only the trace points of the rclcpp layer are available for generic_publish().

Reply:
I thought that the dds_bind_addr_to_stamp trace point was not shown in cyclonedds and humble until you add it to rcl_publish_serialised_message() even if you do generic_publish(). I thought that would be added after jazzy, but am I wrong?
For fastdds, the dds_bind_addr_to_stamp trace point does appear, right?
Separately, did the dds_bind_addr_to_addr trace point actually output?

Reply:
In generic_publish(), dds_bind_addr_to_stamp should be output, so this ‘Info’ is dropped.